src/styles/card.scss

```diff
@@ -0,0 +1,103 @@
+@use './variables' as *;
+
+.product {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 0 16px;
+
+  &__main-image {
+    margin: 32px auto 40px;
+  }
+
+  &__item--name {
+    margin-bottom: 4px;
+  }
+
+  &__item--description {
+    margin-bottom: 16px;
+  }
+
+  &__button {
+    margin-top: 16px;
+    margin-bottom: 16px;
+  }
+}
+
+.main-image {
+  background-image: url(../images/imac.jpeg);
+  width: 160px;
+  height: 134px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: 100% 100%;
+}
+
+.wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.item {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+
+  &--name {
+    font-weight: 500;
+    color: $text-main-color;
+  }
+
+  &--description {
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-secondary-color;
+  }
+
+  &--views {
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-main-color;
+    margin-top: 2px;
+  }
+
+  &--price {
+    color: $text-secondary-color;
+    padding-top: 24px;
+  }
+
+  &--amount {
+    font-size: 16px;
+    font-weight: 700;
+    color: $text-main-color;
+    padding-top: 24px;
+  }
+}
+
+.button {
```

Comment:
The class `.button` should follow the BEM naming convention if it's related to `.product`. If it represents a button inside a product card, it should be named something like `.product__button` to maintain consistency with the BEM structure.